app/views/admin/questions/_form.html.slim

```diff
@@ -12,8 +12,21 @@
     = render 'layouts/messages'
     = f.label :topic_id, "Topic"
     = f.collection_select(:topic_id, Topic.all, :id, :title, {:prompt => 'Select a topic', :disabled => 'Select a topic'}, {class: 'form-control'})
+    = f.label 'Level'
+    .form-group
+      .btn-group data-toggle="buttons"
+        - ['beginner', 'good', 'strong'].each do |val|
+          - if @question.level == val
+            = f.label :level, value: val, class: 'btn btn-default active' do
+              = f.radio_button :level, val
+              | #{val}
+          - else 
+            = f.label :level, value: val, class: 'btn btn-default' do
+              = f.radio_button :level, val
+              | #{val}    
     = f.label :content, 'Question'
     = f.text_area :content, :placeholder => "Write a question", class: 'form-control', rows: 6
     = f.label :answer
     = f.text_area :answer, :placeholder => "Write an answer", class: 'form-control', rows: 6
     = f.submit yield(:button_text), class: "btn btn-md btn-success col-md-4 col-md-offset-4"
+= javascript_include_tag "application"
```

Comment:
I know that we don't need to include application.js but without this line of code it doesn't work.